contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -425,6 +419,20 @@ variable like this:
 This way, we won't have to manually set the directory location
 each time we want to reference it.
 
+You also need to add ``$RISCV_TOOLCHAIN_PATH/bin`` to your user's PATH environment variable.
+
+::
+
+	PATH="$RISCV_TOOLCHAIN_PATH/bin:$PATH"
```

Comment:
Setting this will only work if it's a part of the command as a prefix to it. Otherwise, it will only be changed in the shell itself. https://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export

The command below should be edited to include this, it doesn't need its own separate code block.

Reply:
Fixed.